gen/Project.toml

```diff
@@ -0,0 +1,8 @@
+[deps]
+Clang = "40e3b903-d033-50b4-a0cc-940c62c95e31"
```

Comment:
You should specify a Clang version in compat. Having a Manifest.toml wouldn't be a bad idea either